docs/configuration/module-config/serial.mdx

```diff
@@ -294,3 +294,170 @@ previousState = currentState; // update the previous state
 
 }
 ```
+### Interfacing with button an relay With External Microcontroller
+
+Here are two different examples that work together or separately.
+In the first code when the button is pressed the word "relay1toggle" will be sent via serial which will turn on or off the relay that is connected in the example below.
+
+In the second code, the relay will be activated when t-beam receives the word "relay1toggle" or deactivated depending on its status.
```

Comment:
Maybe:
In the second example, when the t-beam receives the word "relay1toggle", the relay will be activated, or deactivated, depending on it's state.